### ISSUE TITLE
Move messages not working in threaded view tab

### ISSUE THREAD

**Issue by Steff-n** [opto/nostalgy-xpi#42]
Hi,

I think this has been an issue with the previous version of Nostalgy already, but equally applies to Nostalgy++ : 

One of TB's features I use most often is the "show in thread" option from the right-click menu in TB's message list pane - this opens the message together with all related messages (from the same thread) in a new tab, which is very useful to trace a message back to its origin and see the full thread with all related messages in a list view, no matter where (in which folder) the individual messages are currently located. From there, I can easily move the entire thread to some specific folder using Nostalgy. 

However, when the "_Restrict folder completion to the current server_" option from Nostalgy's "_Preferences -> Completion_" tab is activated, this workflow will not work anymore: The "_Move messages to:_" field will not allow to query & auto-complete folder names anymore - I can type whatever I want, the field just doesn't return any folder suggestions. After ticking off the named option from the Preferences, everything works fine again in the extra tab with the threaded messages list. 

This is somewhat annoying as I have a lot of local Archive folders and also folder with similar or identical names across different mailboxes which hence get displayed in every query unless I restrict the query to the current server. 

So being able to restrict Nostalgy++ queries to the current server no matter in which tab or pane I am, would be a highly appreciated improvement. 

Thanks for checking into it!

Steff-n

**Comment by opto** [comment]
Here, it puts this view into a new tab. Probably, some code functions are not available there.
This might be a starting point to solve thhis sometime.

**Comment by Steff-n** [comment]
Yes, that's exactly the point I think: the right-click menu option puts the threaded view into a new tab, and in that tab the _Restrict folder completion to the current server_ setting prevents the code from being executed correctly in that tab.
